script.js

```diff
@@ -34,3 +35,234 @@ link2.addEventListener('click', () => {
+        <li class="btn-2"><button>bootstrap</button></li>
+        <li class="btn-2"><button>Ruby</button></li>
+    </ul>
+    <div class="btn-container">
+        <button class="btn-3 open">See Project</button>
+    </div>
+    </div>
+    <div class="projects">
+    <h4 class="header-4">Professional Art Printing Data</h4>
+    <p class="project-details">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+    <ul class="language" id="tech-btn">
+        <li class="btn-2"><button>html</button></li>
+        <li class="btn-2"><button>bootstrap</button></li>
+        <li class="btn-2"><button>Ruby</button></li>
+    </ul>
+    <div class="btn-container">
+        <button class="btn-3 open">See Project</button>
+    </div>
+    </div>
+    <div class="projects">
+    <h4 class="header-4">Professional Art Printing Data</h4>
+    <p class="project-details">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+    <ul class="language" id="tech-btn">
+        <li class="btn-2"><button>html</button></li>
+        <li class="btn-2"><button>bootstrap</button></li>
+        <li class="btn-2"><button>Ruby</button></li>
+    </ul>
+    <div class="btn-container">
+        <button class="btn-3 open">See Project</button>
+    </div>
+    </div>
+    <div class="projects">
+    <h4 class="header-4">Professional Art Printing Data</h4>
+    <p class="project-details">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+    <ul class="language" id="tech-btn">
+        <li class="btn-2"><button>html</button></li>
+        <li class="btn-2"><button>bootstrap</button></li>
+        <li class="btn-2"><button>Ruby</button></li>
+    </ul>
+    <div class="btn-container">
+        <button class="btn-3 open">See Project</button>
+    </div>
+    </div>
+`;
+
+const obj = [{
+  name: ['Multi-Post Stories', 'Keeping Track of Hundreds of components website'],
+  description: ["Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.", "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"],
+  img: 'images/project-background.png',
+  technologies: ['html', 'bootstrap', 'Ruby on Rails'],
+  live: 'https://amrit131.github.io/portfolio-mobile/',
+  source: 'https://github.com/Amrit131/portfolio-mobile',
+}];
+
```

Comment:
- Currently, your site is populating your popup with the same details when I click on all the cards, this is majorly because you have included just one object in your project list, I'd suggest you add more objects containing details of different projects and display the correct details when the corresponding button is clicked. I see you have 7 project cards, so you could add details for 7 different projects. This would make it easier for you to add more projects in the future 😄 😄 . Please fix this 👍 👍

Reply:
I have implemented the said feedback.